docs/examples/docker-compose.truenas.yml

```
@@ -89,7 +81,25 @@ services:
       interval: 10s
       timeout: 5s
       retries: 5
-    volumes:
+      dramatiq_worker:
+    image: elerevil/fo-shelter-be:${BE_VERSION:-latest}
```

YAMLlint (1.38.0)

[error] duplication of key "image" in mapping

(key-duplicates)